### PR TITLE
Create/mimi integration platform index

### PR DESCRIPTION
**Pull request by MimiDumpling**
Brand new content on how to install and configure integrations on Sentry's new integration platform.

### CODE REVIEW

File: src/collections/_documentation/workflow/integrations/integration-platform/index.md [hunk 1]
```diff
@@ -0,0 +1,237 @@
+---
+title: Integration Platform
+sidebar_order: 2
+---
+# Sentry Integration Platform
+
+Sentry's Integration Platform provides a way for external services to interact with the Sentry SaaS service using the REST API and webhooks. Integrations utilizing this platform are first class actors within Sentry and begin in an unpublished state where the app is available for use in your organization. 
+
+For differences between an Integration Platform Application and prior methods of integrating with Sentry, see the FAQ at the bottomo of the page.
```
Comment (MeredithAnya) [inline src/collections/_documentation/workflow/integrations/integration-platform/index.md]: `bottomo` hehe

Comment (MimiDumpling) [reply src/collections/_documentation/workflow/integrations/integration-platform/index.md]: LOL :)

File: src/collections/_documentation/workflow/integrations/integration-platform/index.md [hunk 2]
```diff
@@ -0,0 +1,237 @@
...
+    data = resp.json()
+    
+    access_token = data['token']
+    refresh_token = data['refreshToken']
+    # ... Securely store the install_id, access_token and refresh_token in DB ...
+    
+    return redirect('http://sentry.io/settings/')
+```
+
+Next, implement the Webhook URL endpoint, `/webhook`, which receives requests based on your resource subscriptions. In this case that includes when a issue is created. In this method, you'll use the Sentry API to check if the issue belongs to a project called Backend, and if it does, you'll forward the issue to Pushover.
```
Comment (MeredithAnya) [inline src/collections/_documentation/workflow/integrations/integration-platform/index.md]: a issue --> an issue

Comment (MimiDumpling) [reply src/collections/_documentation/workflow/integrations/integration-platform/index.md]: nice catch ;)

**Pull request merged**